src/transactions/signed/RemovePublicAddress.ts

```diff
@@ -0,0 +1,37 @@
+import { PublicAddress } from '../../entities/PublicAddress'
+import { SignedTransaction } from './SignedTransaction'
+import { Constants } from '../../utils/constants'
+import { validationRules } from '../../utils/validation'
+
+export class RemovePublicAddress extends SignedTransaction {
```

Comment:
maybe it also should has `RemovePublicAddresses` name

Reply:
yes, ill fix this